<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWin::Internal::MainWindowPreferencesWidget</class>
 <widget class="QWidget" name="MainWin::Internal::MainWindowPreferencesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Update checking</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="updateCheckingCombo">
     <item>
      <property name="text">
       <string>At startup</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Each weeks</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Each months</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Each quarters</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Never (on user's demand)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
